datatypes/src/raster/grid.rs

```diff
@@ -319,6 +319,29 @@ where
     pub fn inner_ref(&self) -> &Vec<T> {
         &self.data
     }
+
+    /// reverse this grid along the y-axis. Returns an "up-side-down" `Grid`.
+    #[must_use]
+    pub fn reversed_y_axis_grid(&self) -> Grid<D, T>
```

Comment:
Are there also files where the x axis is inverted in the tiff? Then we would need a method to invert that as well

Reply:
i have not seen that. However we can add it the same way if we see some in the wild.